app/views/notifications/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing Notification</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Show', @notification %> |
```

Comment:
If you could please remove these links, as we use a sidebar for navigation, in this style
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/views/certs/edit.html.erb#L3